src/backend/distributed/planner/distributed_planner.c

```diff
@@ -559,7 +559,22 @@ int
 GetRTEIdentity(RangeTblEntry *rte)
 {
 	Assert(rte->rtekind == RTE_RELATION);
-	Assert(rte->values_lists != NIL);
+
+	/*
+	 * RangeTblEntry value_lists must not be empty at this point.
+	 * However, this might happen when processing SQL functions
+	 * in which case we will throw an error.
+	*/
+	if (rte->values_lists == NIL)
```

Comment:
I think we can drop below assertion as well and do:

```suggestion
	if (list_length(rte->values_lists) != 2)
```

Reply:
Changed to suggested.